.dockerignore

```diff
@@ -0,0 +1,7 @@
+/packages/
+/build/
+/cache/
+/docs/
+/config.toml
```

Comment:
Did you mean config.toml.example ?

Reply:
Nop.

When using `nbuild` in an unprotected environment, users will most likely copy `config.toml.example` into `config.toml` to use `nbuild`.

I wanna make sure their configuration doesn't end up in the repository.